bower.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "edx-ui-toolkit",
+  "homepage": "https://github.com/edx/edx-ui-toolkit",
+  "private": false,
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components"
+  ],
+  "dependencies": {
+    "requirejs-plugins": "~1.0.3",
+    "jquery": "~1.11.1",
+    "underscore": "~1.8.2",
+    "backbone": "~1.1.2",
+    "requirejs": "~2.1.15",
+    "jasmine": "~2.0.4"
```

Comment:
NIT: This may be an ignorant designer speaking, but is this item more of a devDependency or something you'd use in production too?